packages/plugins/rubygem-hammer_cli_foreman_kubevirt/rubygem-hammer_cli_foreman_kubevirt.spec

```diff
@@ -77,19 +53,22 @@ install -m 0644 .%{gem_instdir}/config/%{plugin_name}.yml \
 
 %files
 %dir %{gem_instdir}
+%license %{gem_instdir}/LICENSE
 %{gem_libdir}
+%{gem_instdir}/locale
```

Comment:
this folder doesn't exist in hammer_cli_foreman_kubevirt-0.1.5 and thus breaks the build